api/kyverno/v1/common_types.go

```diff
@@ -86,6 +86,9 @@ type ContextEntry struct {
 	// ConfigMap is the ConfigMap reference.
 	ConfigMap *ConfigMapReference `json:"configMap,omitempty" yaml:"configMap,omitempty"`
 
+	// Default is a default value which is used when failed to fetch data from APICall.
+	Default string `json:"default,omitempty" yaml:"default,omitempty"`
```

Comment:
If it's only the default value for `apiCall`, should we add to https://github.com/kyverno/kyverno/pull/9879/files#diff-96aa4a3bdb8699fdb9f700c9554e061ebd9578d0c9ea69654a8ecb20d13e3cd8R94?

Reply:
Do we expect this as a string or a genaral JSON format?

Reply:
> If it's only the default value for `apiCall`, should we add to https://github.com/kyverno/kyverno/pull/9879/files#diff-96aa4a3bdb8699fdb9f700c9554e061ebd9578d0c9ea69654a8ecb20d13e3cd8R94?

From the https://github.com/kyverno/kyverno/issues/9723, what i understood is we need to set default value when we fail fetch data from apiCall. What I thought is that if we get error form apiCall we can use default and continue with it ...

Reply:
> Do we expect this as a string or a genaral JSON format?

In https://github.com/kyverno/kyverno/issues/9723 it is mentioned to have default as string. Should we need to use JSON intsead?